content/ngf/how-to/data-plane-configuration.md

```diff
@@ -282,11 +278,13 @@ Status:
 
 If everything is valid, the `ResolvedRefs` condition should be `True`. Otherwise, you will see an `InvalidParameters` condition in the status.
 
+{{< note >}} The `NginxProxy` resource must reside in the same namespace as the Gateway it is attached to. {{< /note >}}
+
 ---
 
 ## Configure the data plane log level
 
-You can use the `NginxProxy` resource to dynamically configure the Data Plane Log Level.
+You can use the `NginxProxy` resource at the GatewayClass level to dynamically configure the log level for all data plane instances.
```

Comment:
This change isn't necessary. We don't need to talk about "globally" changing it. It can be changed at either global or per-Gateway.